### PR TITLE
Just minor orthography improvements

### PR DESCRIPTION
**Pull request by steampilot**

### CODE REVIEW

File: en/core-utility-libraries/xml.rst [hunk 1]
```diff
@@ -255,8 +255,7 @@ Creating a child
 ----------------
 
 The Xml class of CakePHP 2.0 doesn't provide the manipulation of content, this
-must be made using SimpleXMLElement or DOMDocument. But, how CakePHP is so
-sweet, below has the steps to do for create a child node::
+must be done using SimpleXMLElement or DOMDocument. But sweet CakePHP got you covered. Below you see the steps for creating a child node with CakePHP::
```
Comment (ravage84) [inline en/core-utility-libraries/xml.rst]: Line, too long

Comment (dereuromark) [inline en/core-utility-libraries/xml.rst]: We could probably remove the word sweet ;)

File: en/core-utility-libraries/xml.rst [hunk 2]
```diff
@@ -3,20 +3,20 @@ Xml
 
 .. php:class:: Xml
 
-The Xml class was all refactored. As PHP 5 has
+The Xml class was all refactored. PHP 5 implements a
 `SimpleXML <http://php.net/simplexml>`_ and
-`DOMDocument <http://php.net/domdocument>`_, CakePHP doesn't need to
+`DOMDocument <http://php.net/domdocument>`_, so that CakePHP don't need to
```
Comment (markstory) [inline en/core-utility-libraries/xml.rst]: This should stay as doesn't.

File: en/core-utility-libraries/xml.rst (hunk 1 above)
Comment (ADmad) [inline en/core-utility-libraries/xml.rst]: "But CakePHP has you covered."

**Comment by cake17** [comment]
I merge this PR and will make a commit with your comments.

Thanks for your help @steampilot

**Pull request merged**